### DISCUSSION TITLE
MX switches

### DISCUSSION THREAD

**Discussion by neal-p** [christianselig/caldera-keyboard#9] [Q&A]
Hi! I'm hoping to modify your (amazing) design to use cherry mx switches. I *think* the main change should be in ergogen config, but I'm new to this so might not understand some other consequences, any tips?

I'm planning on converting all of the references to `cx`/`cy` in the `ergogen/config.yaml` to `u` for the standard mx sizes. As well as specifying the `mx` footprint instead of `choc`. I know mx switches are a tad bigger, so wondering if you expect I would need to make any other adjustments? I'm not too sure how to tell when components are overlapping...

I attached the modified `config.yaml` and here's a visual comparison of the PCBs! 
Thanks!

![image](https://github.com/user-attachments/assets/32643110-6692-4c2a-91c0-a6c6c9f7cac7)

[config.txt](https://github.com/user-attachments/files/17967666/config.txt)

**Comment by christianselig** [comment]
Hey! Yeah I believe that's all you should have to do! The main thing will just be making sure everything lines up/fits but I *think* you've mostly got it, I can't think of anything else at least haha. Let me know how it goes!

**Comment by bmpitzen** [comment]
@neal-p did you ever complete the build? Were you able to use the mx switches?

**Comment by crtgregoric** [comment]
I'm also interested in MX switches. @neal-p Did you manage to finish the build?

**Comment by neal-p** [comment]
@crtgregoric @bmpitzen sorry for the incredibly late reply... I ended up going with a board already set up for mx switches since I hadn't even soldered in years and didn't want to give myself so many different places to mess up my first try building 🙃 would love to see if either of you try, still on my list to come back to some day!
